### PR TITLE
RAC-213: add job stop ACL

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/config/acl.yml [hunk 1]
```diff
@@ -121,3 +121,8 @@ pim_enrich_job_tracker_view_all_jobs:
     label: pim_enrich.acl.job_tracker.view_all_jobs
     group_name: pim_enrich.acl_group.job_tracker
     enabled_at_creation: false
+
+pim_importexport_import_stop_job:
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/config/acl.yml:125]:
```suggestion
pim_importexport_stop_job:
```

Also changed in src/Akeneo/Platform/Bundle/ImportExportBundle/Normalizer/JobExecutionNormalizer.php and in src/Akeneo/Platform/Bundle/ImportExportBundle/Datagrid/JobExecutionResultRecordHydrator.php

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Datagrid/JobExecutionResultRecordHydrator.php [hunk 2]
```diff
@@ -33,7 +36,8 @@ public function hydrate($qb, array $options = [])
         foreach ($qb->getQuery()->execute() as $record) {
             $job = $this->registry->get($record['jobName']);
             $recordStatus = new BatchStatus($record['status']);
-            $isStoppable = $recordStatus->isRunning() && $job instanceof StoppableJobInterface && $job->isStoppable();
+            $isGranted = $this->securityFacade->isGranted('pim_importexport_import_stop_job');
+            $isStoppable = $recordStatus->isRunning() && $job instanceof StoppableJobInterface && $job->isStoppable() && $isGranted;
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Bundle/ImportExportBundle/Datagrid/JobExecutionResultRecordHydrator.php:40]: I'm not really confortable by checking granted here and in the normalizer, why not checking this in front ? In others used case it's always how we do no ?

**Review by StevenVAIDIE — COMMENTED** [review]
You should also add validation in : 
src/Akeneo/Platform/Bundle/ImportExportBundle/Controller/Ui/JobTrackerController.php:stopJobAction

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Datagrid/JobExecutionResultRecordHydrator.php (hunk 2 above)
Comment (juliensnz) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Datagrid/JobExecutionResultRecordHydrator.php:40]: IMHO, frontend validation alone is not sufficient. Is there any reason why you don't like the fact that we check here?

**Review by SamirBoulil — APPROVED** [review]

**Review by StevenVAIDIE — APPROVED** [review]

**Pull request merged**